release-pipeline.yml

```diff
@@ -0,0 +1,34 @@
+trigger: 
+  - v2 # disable CI build for branch except v2
```

Comment:
Is there a way to just trigger on tags?

Reply:
I'll try some thing to see if it works.  Right now it will kick off a build that build the artifacts, and the release will run, but when there's no tag, it just won't do anything.  

See example:
https://dev.azure.com/msft-vott/VoTT/_releaseProgress?_a=release-environment-logs&releaseId=1&environmentId=1